lms/templates/ccx/enrollment.html

```diff
@@ -84,6 +84,29 @@
         <label for="student-id" class="sr">${_("Enter username or email")}</label>
         <input name="student-id" id="student-id" class="add-field" placeholder="${_("Enter username or email")}" type="text">
         <input name="student-action" class="add" value="Add Student" type="button">
+        <div class="enroll-option">
+            <input type="checkbox" name="auto-enroll" id="auto-enroll" value="Auto-Enroll" checked="yes" aria-describedby="auto-enroll-helper" disabled>
+            <label style="display:inline" for="auto-enroll">${_("Auto Enroll")}</label>
+            <div class="hint auto-enroll-hint">
+              <span class="hint-caret"></span>
+              <p class="text-helper" id="auto-enroll-helper">
+                ${_("If this option is <em>checked</em>, users who have not yet registered for {platform_name} will be automatically enrolled.").format(platform_name=settings.PLATFORM_NAME)}
+                ${_("If this option is left <em>unchecked</em>, users who have not yet registered for {platform_name} will not be enrolled, but will be allowed to enroll once they make an account.").format(platform_name=settings.PLATFORM_NAME)}
+                <br /><br />
+                ${_("Checking this box has no effect if 'Revoke' is clicked.")}
```

Comment:
The `Revoke` is consistent with the `Revoke access` button on the right side of the screenshot. This is borrowed from the membership ui in the instructor dashboard, i.e. 

<img width="849" alt="screen shot 2015-11-20 at 1 16 19 pm" src="https://cloud.githubusercontent.com/assets/430126/11307874/f45b8e2c-8f88-11e5-89d0-28ca67547b8a.png">

We could switch to `unenroll` throughout this ui if you like, but we've been avoiding ui changes.